integration-tests/security/src/test/java/io/quarkiverse/openapi/generator/it/security/OpenWeatherCustomSecurityTest.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkiverse.openapi.generator.it.security;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import javax.inject.Inject;
+
+import org.acme.openapi.weather.customsecurity.api.CurrentWeatherDataWithCustomSecurityApi;
+import org.acme.openapi.weather.customsecurity.model.Model200;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.junit.jupiter.api.Test;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import io.quarkiverse.openapi.generator.it.security.auth.DummyApiKeyAuthenticationProvider;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTestResource(WiremockOpenWeather.class)
+@QuarkusTest
+@org.eclipse.microprofile.rest.client.annotation.RegisterProvider(DummyApiKeyAuthenticationProvider.class)
```

Comment:
Do you really need to annotate the test class with `RegisterProvider`?

Reply:
definitely not, copy paste issue